src/views/list.ejs

```diff
@@ -9,23 +9,135 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
   </head>
   <body>
+  <div id="navigation-style-table">
     <%- include partials/nav %>
+    </div>
     <style>
+
+
+    p {
+    font-weight: 400;
+
+ font-family: Source Code Pro, monospace;
+
+ color: #1f1ff6;
+
+ font-size: 15px;
+
+    }
+
+
+.top-text {
+
+font-family: Roboto Mono, monospace;
+
+font-size: 15px;
+
+font-style: regular;
+
+font-weight: 500;
+
+text-align: left;
+
+color: white;
+
+text-transform: uppercase;
+
+background: #342af1;
+
+}
+
+.link-button{
+
+display: inline-block;
+
+position: relative;
+
+border: none;
+
+box-shadow: 0px 0px 0px 0px #9fb4f2;
+
+font-weight: 400;
+
+font-family: Source Code Pro, monospace;
+
+font-size: 18px;
+
+padding: 10px 123px;
+
+background: #342af1;
+
+color: #ffffff;
+
+}
+
+  .link-button:hover {
```

Comment:
Hey @marcodeanda , could you please format the changed files using Prettier or equivalent

Reply:
Sure I will get to it!